ckanext/dalrrd_emc_dcpr/helpers.py

```diff
@@ -391,5 +391,40 @@ def mod_scheming_flatten_subfield(subfield, data):
     return flat
 
 
+def get_maintenance_custom_other_field_data(data_dict):
+    """
+    the custom field "maintenance" stores "other"
+    options in an __extra structure in the database,
+    in package_extra table, this structure
+    doesn't show up with regular ckan actions like
+    "package_show", "package_search" ..etc., we need
+    to grab it from the database, if other alternatives
+    can be used (e.g. when using the pkg coming with
+    /package/read.html, the whole data shows up)
+    it would be preferable
+    """
+    q = f""" select value from package_extra where package_id='{data_dict["id"]}' AND key = 'maintenance_information'  """
```

Comment:
@Samweli i don't like this, hitting the database to grab a piece of data already fetched, if you have alternatives please make a comment, please check this first to understand the issue https://github.com/ckan/ckan/issues/6875#issuecomment-1237504600